docs/source/llms/transformers/tutorials/translation/component-translation.ipynb

```diff
@@ -0,0 +1,790 @@
+      "text/plain": [
+       "<Experiment: artifact_location='file:///Users/benjamin.wilson/repos/mlflow-fork/mlflow/docs/source/llms/transformers/tutorials/translation/mlruns/996217394074032926', creation_time=1701286351921, experiment_id='996217394074032926', last_update_time=1701286351921, lifecycle_stage='active', name='Translation', tags={}>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# If you are running this tutorial in local mode, leave the next line commented out.\n",
+    "# Otherwise, uncomment the following line and set your tracking uri to your local or remote tracking server.\n",
+    "\n",
+    "# mlflow.set_tracking_uri(\"http://127.0.0.1:8080\")\n",
+    "\n",
+    "mlflow.set_experiment(\"Translation\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Logging the Model with MLflow\n",
+    "\n",
+    "We are now set to log our translation model with MLflow, ensuring its trackability and version control.\n",
+    "\n",
+    "<details>\n",
+    "    <div>\n",
+    "        <h4>Starting an MLflow Run</h4>\n",
+    "        <p>We initiate the logging process by starting an MLflow run. This encapsulates all the model information, including artifacts and parameters, within a unique run ID.</p>\n",
+    "    </div>\n",
+    "    <div>\n",
+    "        <h4>Using <code>mlflow.transformers.log_model</code></h4>\n",
+    "        <p>This function is integral to logging our model in MLflow. It records various aspects of the model:</p>\n",
+    "        <ul>\n",
+    "            <li><strong>Model Pipeline</strong>: The complete translation model pipeline, encompassing the model and tokenizer.</li>\n",
+    "            <li><strong>Artifact Path</strong>: The directory path in the MLflow run where the model artifacts are stored.</li>\n",
+    "            <li><strong>Model Signature</strong>: The pre-defined signature indicating the model's expected input-output formats.</li>\n",
+    "            <li><strong>Model Parameters</strong>: Key parameters of the model, like <code>max_length</code>, providing insights into model behavior.</li>\n",
+    "        </ul>\n",
+    "    </div>\n",
+    "    <div>\n",
+    "        <h4>Outcome of Model Logging</h4>\n",
+    "        <p>Post logging, we obtain the <code>model_info</code> object, which encompasses all the essential metadata about the logged model, such as its storage location. This metadata is vital for future deployment and performance analysis.</p>\n",
+    "    </div>\n",
+    "</details>\n",
+    "<br/>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
```

Comment:
nit: Can we remove this warning? It's shown many times across the notebooks.

Reply:
Yep! I have added warnings silencing for all UserWarnings (pydantic, setuptools) and added the fix for the tokenizer parallelization warning.